build/Templating.targets

```diff
@@ -5,26 +5,16 @@
 
   <Target Name="BuildTemplates" DependsOnTargets="GeneratePropsFiles">
     <PropertyGroup>
-      <GeneratedNoTimestampPackageVersionPropsPath>$(IntermediateDir)dependencies.notimestamp.props</GeneratedNoTimestampPackageVersionPropsPath>
-      <TemplateProjCommmonProperties>
+      <TemplateProjProperties>
         RepositoryRoot=$(TemplatingProjectRoot);
         DotNetRestoreSourcesPropsPath=$(GeneratedRestoreSourcesPropsPath);
+        DotNetPackageVersionPropsPath=$(GeneratedPackageVersionPropsPath);
         BuildNumber=$(BuildNumber);
         Configuration=$(Configuration);
-      </TemplateProjCommmonProperties>
-      <TemplateProjProperties>
-        $(TemplateProjCommmonProperties);
-        DotNetPackageVersionPropsPath=$(GeneratedPackageVersionPropsPath);
-        VersionMetadata=timestamped;
```

Comment:
Just making sure VersionMetadata isn't needed anymore?

Reply:
Correct. Not needed.